tests/test_release_cleanup.py

```diff
@@ -108,6 +109,37 @@ def test_write_to_file(tmpdir):
             assert output.read() == expected_result
 
 
+def test_return_values_of_prettier(tmpdir):
+    os.chdir(tmpdir)
+
+    # GIVEN a yml file that has extra whitespace padded at the end:
+    yml_text = (Path(_get_test_root()) / "data" / "test_repository.yml").read_text(
+        encoding="utf-8"
+    )
+    repo_with_whitespace = "repo_with_whitespace.yml"
+    Path(repo_with_whitespace).write_text(yml_text + "\n", encoding="utf-8")
+
+    # THEN it will fail with exit code 1 if checked:
+    with pytest.raises(SystemExit) as exit_info:
+        main(["prettier", "--check", "--files", repo_with_whitespace])
+        assert exit_info.value.code == 1
+
+    # Redo to ensure the yml file is not reformatted
+    with pytest.raises(SystemExit) as exit_info:
+        main(["prettier", "--check", "--files", repo_with_whitespace])
+        assert exit_info.value.code == 1
+
+    # IF we run without check, the file should be reformatted
+    with pytest.raises(SystemExit) as exit_info:
+        main(["prettier", "--files", repo_with_whitespace])
+        assert exit_info.value.code == 0
+
+    # IF checked again, the file should be ok, proving it was reformatted:
+    with pytest.raises(SystemExit) as exit_info:
+        main(["prettier", "--check", "--files", repo_with_whitespace])
+        assert exit_info.value.code == 0
+
+
```

Comment:
Very good and clear test. Do we really need the capital letters in some words in comments?

Reply:
It is an attempt at writing test comments according to what @eivindjahren has advocated in the Scout slack channel, on "behavioural driven development". The casing of initial words might not be important.

Reply:
Ah, that issue https://github.com/equinor/ert/pull/4429 was closed unmerged.

Reply:
Upper case removed.